test/features/deploy_travel_agency.feature

```diff
@@ -6,15 +6,25 @@ Feature: Deploy Travel agency service and verify its functionality
     Given Namespace is created
     And Kogito Operator is deployed with Infinispan and Kafka operators
     And Install Kogito Data Index with 1 replicas
-    And Deploy service from example file "travelapp-kogito-travel-agency.yaml"
-    And Deploy service from example file "travelapp-kogito-visas.yaml"
-    And Kogito application "kogito-travel-agency" has 1 pods running within 10 minutes
-    And HTTP GET request on service "kogito-travel-agency" with path "travels" is successful within 1 minutes
-    And Kogito application "kogito-visas" has 1 pods running within 10 minutes
-    And HTTP GET request on service "kogito-visas" with path "visaApplications" is successful within 1 minutes
+    And Deploy quarkus example service "kogito-travel-agency/travels" with native disabled and persistence and events and environment variables
+    | MP_MESSAGING_INCOMING_VISASAPPROVED_BOOTSTRAP_SERVERS                   | |
+    | MP_MESSAGING_INCOMING_VISASREJECTED_BOOTSTRAP_SERVERS                   | |
+    | MP_MESSAGING_OUTGOING_VISAAPPLICATIONS_BOOTSTRAP_SERVERS                | |
+    | MP_MESSAGING_OUTGOING_KOGITO_PROCESSINSTANCES_EVENTS_BOOTSTRAP_SERVERS  | |
+    | MP_MESSAGING_OUTGOING_KOGITO_USERTASKINSTANCES_EVENTS_BOOTSTRAP_SERVERS | |
```

Comment:
Could you try without those env vars?
https://github.com/kiegroup/kogito-examples/pull/184
We will have to wait for @DuncanDoyle PR to get merged though.

Reply:
It works, PR updated.